benchmarks/shifts.py

```
@@ -0,0 +1,107 @@
+
+from mlip_arena.flows.conservation import differential_entropy_along_nve_trajectory
+
+
+@flow
+def distribution_shifts(calculator: str | BaseCalculator, calculator_kwargs: dict | None = None):
+    ctx = FlowRunContext.get()
+    parent_task_runner = ctx.task_runner
+
+    work_dir = Path(__file__).parent / "energy_conservation"
+
+    dH, sampled_structures = differential_entropy_along_nve_trajectory.with_options(
```

Ruff (0.15.9)

[warning] Unpacked variable `dH` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)